plugins/PrivacyManager/PrivacyManager.php

```diff
@@ -516,6 +517,7 @@ public function setTrackerCacheSiteAttributes(&$cacheContent, int $idSite): void
     {
         $config = new Config($idSite);
         $config->setTrackerCache($cacheContent);
+        PolicyManager::storePolicySettingValuesInTrackerCache($cacheContent, $idSite);
```

Comment:
Nitpick, but would make it slightly more clear that it's not related to the config stuff.
```suggestion
        
        PolicyManager::storePolicySettingValuesInTrackerCache($cacheContent, $idSite);
```